types.tsx

```diff
@@ -14,6 +14,18 @@ declare global {
   }
 }
 
+export type User = {
+  firstName: String;
+  lastName: String;
+  residentialCollege: String;
+  classYear: Number;
+  username: String;
```

Comment:
a few notes! (mostly based on the things we talked about last time: https://github.com/clothing100/clothing100/wiki/Schema-Model)

* let's make an id so that we can easily reference a user in other objects
* instead of separating all of the garment lists you have here, lets have lists that refer to these objects. for instance a list of items they are selling/renting will capture most of this. and then in terms of what they have sold, I think we can calculate late based on the exchanges that have them listed as a seller/buyer. 

let me know if anything in that wiki page doesn't make sense, but also @edengorevoy and @rosiesr  if yall have any other thoughts!

Reply:
agree that they should be lists of clothing objects! 
although, maybe it would still be useful to capture garments bought/borrowed in the lists? I feel like at some point going through all transactions to calculate them would possibly take a long time compared to how many garments each user has borrowed/sold. but also i'm not sure if it's just better to store less things